app/templates/components/learnergroup-list.hbs

```diff
@@ -1,79 +1,120 @@
               class="clickable enabled"
-              click=(action "startCopy" learnerGroup)
               title=(t "general.copy")
-            }}
+              click=(action "startCopy" learnerGroup)}}
           {{else}}
             {{fa-icon "copy" class="disabled"}}
           {{/if}}
         </td>
       </tr>
-      {{#if (contains learnerGroup learnerGroupsForRemovalConfirmation)}}
+
+      {{#if (contains learnerGroup this.learnerGroupsForRemovalConfirmation)}}
         <tr class="confirm-removal">
           <td colspan="5">
             <div class="confirm-message">
-              {{t
-                "general.confirmRemoveLearnerGroup"
-                subgroupCount=learnerGroup.children.length
-              }} <br>
+              {{t "general.confirmRemoveLearnerGroup"
+                subgroupCount=learnerGroup.children.length}} <br>
               <div class="confirm-buttons">
-                <button {{action remove learnerGroup}} class="remove text">{{t "general.yes"}}</button>
-                <button {{action "cancelRemove" learnerGroup}} class="done text">{{t "general.cancel"}}</button>
+                <button
+                  class="remove text"
+                  onclick={{action @remove learnerGroup}}>
+                  {{t "general.yes"}}
+                </button>
+                <button
+                  class="done text"
+                  onclick={{action "cancelRemove" learnerGroup}}>
+                  {{t "general.cancel"}}
+                </button>
               </div>
             </div>
           </td>
         </tr>
       {{/if}}
-      {{#if (contains learnerGroup learnerGroupsForCopy)}}
+
+      {{#if (contains learnerGroup this.learnerGroupsForCopy)}}
         <tr class="confirm-copy">
           <td colspan="5">
             <div class="confirm-buttons">
-              <button {{action "copy" true learnerGroup}} class="done text">{{t "general.copyWithLearners"}}</button>
-              <button {{action "copy" false learnerGroup}} class="done text">{{t "general.copyWithoutLearners"}}</button>
-              <button {{action "cancelCopy" learnerGroup}} class="cancel text">{{t "general.cancel"}}</button>
+              <button
+                class="done text"
+                onclick={{action "copy" true learnerGroup}}>
+                {{t "general.copyWithLearners"}}
+              </button>
+              <button
+                class="done text"
+                onclick={{action "copy" false learnerGroup}}>
+                {{t "general.copyWithoutLearners"}}
+              </button>
+              <button
+                class="cancel text"
+                onclick={{action "cancelCopy" learnerGroup}}>
+                {{t "general.cancel"}}
+              </button>
             </div>
           </td>
         </tr>
       {{/if}}
+    {{else}}
+      <tr>
+        <td colspan="6" align="center">
+          {{if @query (t "general.noResultsFound") (t "general.none")}}
```

Comment:
i don't see this being triggered at all.

![Selection_419](https://user-images.githubusercontent.com/1410427/55743400-0eac4880-59e7-11e9-8f8d-526df5e3ca57.png)

![Selection_422](https://user-images.githubusercontent.com/1410427/55743450-30a5cb00-59e7-11e9-9847-725de306a428.png)

Reply:
scratch that, i was looking at stale code. sorry about the confusion!